docs/mobile-apps/automated-testing/appium/quickstart.md

```diff
@@ -54,16 +54,16 @@ export SAUCE_ACCESS_KEY="your Sauce access key"
 
 ## Step 3: Upload the Sample App
 
-1. [Download the iOS Real Device Swag Labs App](https://github.com/saucelabs/sample-app-mobile/releases/download/2.7.1/iOS.RealDevice.SauceLabs.Mobile.Sample.app.2.7.1.ipa).
+1. [Download the iOS Real Device Swag Labs App](https://github.com/saucelabs/my-demo-app-rn/releases/download/v1.3.0/iOS-Real-Device-MyRNDemoApp.1.3.0-162.ipa).
 2. Upload it to your Sauce Labs account, either through the Sauce Labs UI or by calling the API.
 
-- Sauce Labs UI: From [Sauce Labs Home > LIVE > Mobile App](https://app.saucelabs.com/live/app-testing), select **Upload App** and browse to find the `iOS.RealDevice.SauceLabs.Mobile.Sample.app.2.7.1.ipa` file you just downloaded.
+- Sauce Labs UI: From [Sauce Labs Home > LIVE > Mobile App](https://app.saucelabs.com/live/app-testing), select **Upload App** and browse to find the `iOS-Real-Device-MyRNDemoApp.1.3.0-162.ipa` file you just downloaded.
```

Comment:
Updates link to UI -- [App Management](https://docs.saucelabs.com/mobile-apps/live-testing/live-mobile-app-testing/#uploading-an-app) is now used to upload apps.

```suggestion
- Sauce Labs UI: From [Sauce Labs > App Management](https://app.saucelabs.com/app-management), select **Upload App** and browse to find the `iOS.RealDevice.SauceLabs.Mobile.Sample.app.2.7.1.ipa` file you just downloaded.
```